src/equations/equations.jl

```diff
@@ -16,6 +16,21 @@ function Base.show(io::IO, equations::AbstractEquations)
   end
 end
 
+function Base.show(io::IO, ::MIME"text/plain", equations::AbstractEquations)
+  if get(io, :compact, false)
+    show(io, equations)
+  else
+    summary_header(io, get_name(equations))
+    summary_line(io, "#variables ", nvariables(equations))
```

Comment:
```suggestion
    summary_line(io, "#variables", nvariables(equations))
```
Otherwise, we get `#variables :` instead of `#variables:` in the output.

Reply:
Thanks.